CONTRIBUTING.md

```diff
@@ -494,20 +494,22 @@ npx tap --ts --jobs=1 --timeout=600 \"./\"
 > Be aware that glob patterns need quoting in some operating system's shell environments and not necessarily on others.
 > In the npm scripts Cactus uses we quote all of them to ensure a wider shell compatibility.
 
-### Building the SDK
+### Building the API Client
```

Comment:
@Leeyoungone 

```suggestion
### Building the API Client(s)
```

(because when we originally wrote this documentation there was going to be only one API client but then we ended up creating many)

Reply:
Thank you for making suggestions for the documentation 😭 This might have been the hardest PR I've picked up because I just don't know what I was suppose to write.